package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@blablacar/ui-library",
-  "version": "0.24.1",
+  "version": "1.0.0",
```

Comment:
Should we update the Readme by making clear this is still rather an alpha version?

Reply:
Why is it an alpha version ?

Reply:
I think we're missing to many "guidelines" and explanations (and support) to leave this out in the wild.

Reply:
Ok as discussed, adding "guidelines" is planned.